packages/python-pulpcore/python-pulpcore.spec

```diff
@@ -166,17 +166,21 @@ do
   install -D -m 755 ${wrapper} %{buildroot}%{_libexecdir}/%{pypi_name}/${wrapper}
 done
 
+mv %{buildroot}%{_bindir}/pulpcore-worker %{buildroot}%{_libexecdir}/%{pypi_name}/
```

Comment:
Should we use `install -D -m 755` to ensure permissions?

Reply:
it is in bindir, so I would expect it to already have correct perms.

Reply:
I went with wrapper for consistency